tests/sentry/explore/endpoints/test_explore_saved_queries.py

```diff
@@ -222,6 +222,27 @@ def test_get_expired_query(self):
         assert response.status_code == 200, response.content
         assert response.data[0]["expired"]
 
+    def test_get_my_queries(self):
+        with self.feature(self.feature_name):
+            response = self.client.get(self.url, data={"exclude": "shared"})
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 1
```

Comment:
Should we assert the `id` because both tests here check that the `response.data` has one item, we don't really have guarantee that they're not the same item

Reply:
Updated to check for `name`!